partcad/src/partcad/part_factory_scad.py

```
@@ -86,3 +89,108 @@ async def instantiate(self, part):
+            return
+
+        if args:
+            args_str = ", ".join(f"{k}={v}" for k, v in args.items())
+
+        new_line = f"{method}({args_str});"
+
+        base_dir = os.path.dirname(self.path)
+        if not base_dir:
+            base_dir = "."
+
+        lock_path = "/tmp/partcad.lock"
```

Ruff (0.8.2)

Probable insecure usage of temporary file or directory: "/tmp/partcad.lock"

(S108)